src-internal/Admin/WcPayWelcomePage.php

```diff
@@ -72,7 +72,7 @@ public function register_payments_welcome_page() {
 				'admin.php?page=wc-admin&path=/wc-pay-welcome-page',
 				null,
 				$menu_icon,
-				55.7,
+				56,
```

Comment:
Maybe try `57` instead?

Reply:
Looks like `57` will make the menu item go down and `55` will somehow make it before WooCommerce. 

So I changed `Analytics`  position to `57` instead in https://github.com/woocommerce/woocommerce-admin/pull/8399/commits/aa888ed1efbed9a0ccebeeb5d06d0cb52e39a46f.

- `WooCommerce`: `55`
- `Products`: `55`
- `Payments `: `56`
- `Analytics`: `57` <-- changed
- `Marketing`: `58`

Results:

![Screen Shot 2022-03-07 at 14 39 26](https://user-images.githubusercontent.com/4344253/156980826-f47d5dbe-c176-40e5-808e-1fb65f305762.png)